core/controlplane/config/templates/cloud-config-controller

```diff
@@ -987,6 +987,12 @@ write_files:
 
       applyall "/srv/kubernetes/manifests/kube-system-ns.yaml"
 
+      # Daemonsets
+      applyall \
+        "${mfdir}/kube-proxy-ds.yaml" \
+        {{ if .Experimental.NodeDrainer.Enabled }}"${mfdir}/kube-node-drainer-ds.yaml"{{ end }} \
```

Comment:
I had been a bit nervous about moving kube-node-drainer-ds altogether, wondering if `kube-node-drainer` would depend on cluster dns. But actually we had been [deploying the drainer before dns](https://github.com/kubernetes-incubator/kube-aws/blob/c6d72bc9ad927b5509932bb55d614e69d02e19b5/core/controlplane/config/templates/cloud-config-controller#L1080-L1089). No problem, then!